xml/System.Xml/XmlElement.xml

```diff
@@ -733,11 +733,11 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- Setting this property replaces all the children with the parsed contents of the given string.  
+
+Setting this property replaces all the children with the parsed contents of the given string, so children nodes are not preserved.
```

Comment:
I'd put that in the brackets or as another sentence. Maybe even connect it with the next sentence and put it "Remarks" or something like that